transmission/userSetup.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+
+# More/less taken from https://github.com/linuxserver/docker-baseimage-alpine/blob/3eb7146a55b7bff547905e0d3f71a26036448ae6/root/etc/cont-init.d/10-adduser
+
+RUN_AS=root
+
+if [ -n "$PUID" ] && [ ! "$(id -u root)" -eq "$PUID" ]; then
+    RUN_AS=abc
+    if [ ! "$(id -u ${RUN_AS})" -eq "$PUID" ]; then usermod -o -u "$PUID" ${RUN_AS} ; fi
+    if [ ! "$(id -g ${RUN_AS})" -eq "$PGID" ]; then groupmod -o -g "$PGID" ${RUN_AS} ; fi
+
+    echo "Setting owner for transmission paths to ${PUID}:${PGID}"
+    chown -R ${RUN_AS}:${RUN_AS} ${TRANSMISSION_HOME}
+    chown ${RUN_AS}:${RUN_AS} \
```

Comment:
Hmm. Maybe. Then again, the /config folder basically just contains username and passwords for openvpn and transmission web ui. As long as it can read the file, it doesn't have to own it. Don't think it would change too much one way or another.